src/dispatch/entity_type/service.py

```diff
@@ -105,6 +117,12 @@ def update(
 
     entity_type.signals = signals
 
+    try:
+        json.loads(entity_type_in.jpath)
```

Comment:
I believe this will always raise an exception, as `JSONPath` is not valid `JSON`, e.g. `$.user.hostname` would be a valid `JSONPath`

Reply:
Fixed.

Reply:
Thanks, that looks good. The only thing remaining I think would be valuable to add is a unit test that ensures this library is not vulnerable to the same issue in the javascript version and does indeed resolve the issue. Might want to move the parsing to a dedicated function to make testing easier.

Apologies for not calling all this out up front.

Reply:
Done.